case_studies/coadds/signal_noise_ratio.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+import torch
+
+
+def n_s_ratio(img, background):
+    """Calculate Signal-to-Noise Ratio.
+    Args:
+        img: Tensor of shape `(N x C x H x W)` where N is the batch_size, C is the \
```

Comment:
you don't need the `\` at the end I don't think (since the documentation uses tripe quotes.